builds/serverless-config-pack/serverless-go.yml

```diff
@@ -102,24 +102,26 @@ functions:
 #Start:resources
 resourcesDisabled:
   Resources:
+    #Start:customRoles
     #Start:isVaultEnabled
```

Comment:
Now that we want to keep this agnostic of Vault feature being enabled or not, I would remove the word here. Instead I prefer using `isBasicRole`, `isVpcEnabled` instead of `isVaultEnabled` & `isVaultAndVpcEnabled`..

Reply:
Renamed